test-suite/src/test/require-jdk7/org/scalajs/testsuite/javalib/util/CollectionsTestOnJDK7.java

```diff
@@ -5,12 +5,14 @@
 ** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
 **                          |/____/                                     **
 \*                                                                      */
-package org.scalajs.testsuite.javalib
+package org.scalajs.testsuite.javalib.util
 
 import java.{util => ju}
```

Comment:
I have no idea. I made sure that the character where exactly the same as the ones that were there before the commit.